test/msw-api/setup-worker/start/on-unhandled-request/error.mocks.ts

```diff
@@ -1,7 +1,10 @@
 import { setupWorker, rest } from 'msw'
 
 const worker = setupWorker(
-  rest.get('/user', (req, res, ctx) => {
+  rest.get('/user/:id', (req, res, ctx) => {
```

Comment:
I think we should add a new handler just for this particular scenario. It's always better to keep them separated: easier to reason and refactor. 

```js
rest.get('/user') // leave the previous handler
rest.get('/new-name') // a new handler for this use case
```